addons/interact_menu/stringtable.xml

```diff
@@ -489,5 +489,9 @@
             <Spanish>Selector de color</Spanish>
             <Turkish>Seçici Renk</Turkish>
         </Key>
+        <Key ID="STR_ACE_Interact_Menu_combineParentWithSingleChild">
+            <English>Combine parent menu with single child item</English>
```

Comment:
I think title case for setting display names, maybe something like: "Consolidate Single Child Actions". And then the description can be: "Combines parent actions with only one child action together."

Reply:
> "Combines parent actions with only one child action together."

I don't think camel case is good for settings especially for those looking like sentences. Also I replaced `parent actions` with `parent action`.